Cython/Utility/Builtins.c

```diff
@@ -127,9 +127,9 @@ static PyObject* __Pyx_PyExec3(PyObject* o, PyObject* globals, PyObject* locals)
         // For the limited API we just defer to the actual builtin
         // (after setting up globals and locals) - there's too much we can't do otherwise
         PyObject *builtins, *exec;
-        builtins = PyEval_GetBuiltins();
+        builtins = PyEval_GetBuiltins();  // borrowed
         if (!builtins) return NULL;
-        exec = PyDict_GetItemString(builtins, "exec");
+        exec = PyDict_GetItem(builtins, PYIDENT("exec"));  // borrowed
```

Comment:
If you move the comments into a separate line instead of at the end of a code line, they'll be removed from the generated C files.